templates/cloud9-autodeploy.cfn.yml

```diff
@@ -0,0 +1,375 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: AWS CloudFormation template for various workshops. Creates a Cloud9
+  bootstraps the instance.
+Parameters:
+  WSC9InstanceType:
+    Description: Workshop Cloud9 instance type
+    Type: String
+    Default: t3.small
+    AllowedValues:
+      - t3.small
+    ConstraintDescription: Must be a valid Cloud9 instance type
+  WSC9vpccidr:
+    Description: Workshop Cloud9 VPC CIDR CidrBlock
+    Type: String
+    Default: 10.0.0.0/24
+  WSC9vpcSubNet:
+    Description: Workshop Cloud9 Subnet
+    Type: String
+    Default: 10.0.0.0/25
+  WSC9EnvType: 
+    Description: Environment type.
+    Default: awsaccount
+    Type: String
+    AllowedValues:
+      - awsaccount
+    ConstraintDescription: must specify eventengine or awsaccount.
+Conditions: 
+  CreateEventEngineResources: !Equals [ !Ref WSC9EnvType, eventengine ]
+Resources:
+
+########### NETWORK ############
+  WSC9VPC:
```

Comment:
We should target the existing default gateway, rather than creating yet another VPC